OTAnalytics/plugin_parser/argparse_cli_parser.py

```diff
@@ -27,6 +27,12 @@ def _setup(self) -> None:
             help="Start OTAnalytics CLI. If omitted OTAnalytics GUI will be started.",
             required=False,
         )
+        self._parser.add_argument(
+            "--show_svz",
```

Comment:
Can we rename it to `--show-svz` so that this argument is uniform to the other arguments?

Reply:
Sure, I already changed it.